lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -14,6 +14,12 @@
         platform.localeBestFitCache = new platform.Map();
     }
 
+    // determine what backing library we are using
+    // making these vars in JS allows us to more change how we
+    // determine the backing library
+    let isICU = !platform.winglob;
+    let isWinGlob = platform.winglob;
```

Comment:
nitpick (Don't change anything unless you've already got another change that will cause you to regenerate bytecode anyway)

Maybe be more explicit with these names since they will be unscoped in Intl.js (with anything like `platform.isICU`). Maybe `platform.winglob` was unfortunate naming for a boolean as well, but don't update that.

I'd suggest `isPlatformUsingICU` or something.